src/status_im/components/list/views.cljs

```diff
@@ -97,15 +104,19 @@
 (defn- default-render-section-header [m]
   [rn/text {:style lst/section-header} (:title m)])
 
+(defn- wrap-per-section-render-fn [v]
+  (map #(if-let [f (:render-fn %)] (assoc (dissoc % :render-fn) :renderItem f) %) v))
```

Comment:
For anon function with multiple usage of %, can we give this variable a descriptive name instead? E.g. what is `v` and what is an item in `v`? A bit hard to parse.

Reply:
Sure